platform/lib/platform_web/live/adminland_live/api_token_live.ex

```diff
@@ -63,10 +63,7 @@ defmodule PlatformWeb.AdminlandLive.APITokenLive do
                       </th>
                       <th scope="col" class="py-3.5 pl-3 pr-4 sm:pr-6">
                         <span class="sr-only">More Actions</span>
-                        <%= live_patch("New Token",
-                          class: "button ~urge @high float-right",
-                          to: Routes.adminland_index_path(@socket, :api_new)
-                        ) %>
+                        <.link method = "New Token" class = "button ~urge @high float-right" patch = {Routes.adminland_index_path(@socket, :api_new)}></.link>
```

Comment:
As above, `method` does not apply labels to buttons/links. With this change, the button would not have a label.